x-pack/plugins/enterprise_search/public/applications/app_search/components/api_logs/components/new_api_events_prompt.scss

```diff
@@ -0,0 +1,6 @@
+.newApiEventsPrompt {
+  padding: $euiSizeXS;
+  padding-left: $euiSizeS;
```

Comment:
FYI: without this custom padding, the height of the panel exceeds the height of the row height and causes the page to bounce around awkwardly, so I added it as a polish item